src/components/ListedSpecies/SexSection.jsx

```diff
@@ -0,0 +1,58 @@
+import {
+  FormControl,
+  FormLabel,
+  Grid,
+  GridItem,
+  Input,
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Input' is defined but never used.